### DISCUSSION TITLE
Is there a way to remove Intl.so requirement?

### DISCUSSION THREAD

**Discussion by yabdab** [statamic/cms#4988] [Q&A]
I am using MAMP Pro and the Intl.so extension is not provided.

Is there a way to remove the requirement for Intl.so  when using Statamic?

**Comment by edalzell** [comment]
Nope, are you having difficulty getting it installed or finding a server with it?

**Reply by yabdab** [reply]
Ended up just using Valet instead.  Still working. on it, but I like it a lot so far  😁
